docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -540,64 +547,67 @@ spec:
   - pre-adoption-validation
 EOF
 ----
-+
-Wait for the validation to finish.
+
+.. Wait for the validation to finish.
 
 .. Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
-watch oc get pod -l app=openstackansibleee
+$ watch oc get pod -l app=openstackansibleee
 ----
 +
 ----
 # following the ansible logs with:
-oc logs -l app=openstackansibleee -f --max-log-requests 20
+$ oc logs -l app=openstackansibleee -f --max-log-requests 20
 ----
 
 .. Wait for the deployment to reach the `Ready` status:
 +
 ----
-oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
+$ oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
++
+[IMPORTANT]
+====
+If any openstack-pre-adoption validations fail, you must determine
+which ones were unsuccessful based on the Ansible logs, and then try the
```

Comment:
Would you know which pre-adoption validation(s) are unsuccessful based on the error messages and double check that with the logs?

"If any openstack-pre-adoption validations fail, you can reference the Ansible logs, and then try the following troubleshooting options:"

Reply:
It sounds like a message will indicate that there is an error, and then you use the Ansible logs to determine which ones were unsuccessful. Your rewrite makes more sense.